Prototyper/Classes/Feedback/FeedbackBubble.swift

```diff
@@ -58,4 +59,20 @@ class FeedbackBubble: UIView {
         }
     }
     
+    private func setBubbleImageToButton(button: UIButton) {
+        let backgroundCircle = UIImage(systemName: "circle.fill", withConfiguration: UIImage.SymbolConfiguration(pointSize: 70, weight: .heavy))?.withTintColor(.systemBlue, renderingMode: .alwaysTemplate)
+        let shareIcon = UIImage(systemName: "square.and.arrow.up", withConfiguration: UIImage.SymbolConfiguration(pointSize: 40, weight: .light))?.withTintColor(.white, renderingMode: .alwaysOriginal)
+        guard let backgroundCircleImage = backgroundCircle, let shareIconImage = shareIcon else {
+            return
+        }
+        let feedbackIcon = UIImage.overlapImages(topImage: shareIconImage, bottomImage: backgroundCircleImage)
+        button.setImage(feedbackIcon, for: .normal)
+    }
+    
+    private func addShadow(view: UIView) {
+        view.layer.shadowColor = UIColor.black.cgColor
```

Comment:
I would suggest to use e.g.: `UIColor.systemGray.cgColor` so we have a shadow in dark mode.